src/content/config.ts

```diff
@@ -12,6 +12,23 @@ const sponsorsCollection = defineCollection({
     }),
   ),
 });
+
+const staffCollection = defineCollection({
+  type: "data",
+  schema: z.array(
+    z.object({
+      id: z.number(),
+      name: z.string(),
+      icon: z.string(),
+      organizationName: z.string(),
+      twitterLink: z.string(),
+      githubLink: z.string(),
```

Comment:
この2つはIDだけを入稿して、URLはViewで生成で良いと思います 🙆‍♂️ 
```suggestion
      twitterId: z.string(),
      githubId: z.string(),
```

Reply:
その形にします。